src/TrackGenerator.cpp

```diff
@@ -429,19 +429,19 @@ bool TrackGenerator::containsTracks() {
  *
  * @code
  *          num_tracks = track_generator.getNumTracks()
- *          coords = track_generator.retrieveTrackCoords(num_tracks*4)
+ *          coords = track_generator.retrieveTrackCoords(num_tracks*6)
```

Comment:
Are we always returning 3D coordinates or should the 4 and 6 be replaced with something like NUM_SURFACES